README.md

```diff
@@ -1 +1,110 @@
 # react-overridable
+
+With `react-overridable` you can mark your React components as overridable
+and allow other apps to customize them. This can be useful when creating
+libraries with a default implementation of components which requires to be
+overridden at runtime.
+
+You can inject new props, override render elements or the component itself.
+
+## Usage
+
+Create a React component and mark it as overridable:
+
+```js
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Overridable, {parametrize, OverridableContext} from 'react-overridable';
+
+class TitleComponent extends Component {
```

Comment:
Eventually I'd like the readme to use a function component (and maybe an example with a class component somewhere else). Not super urgent though.

Reply:
I think there is really room for improvements and I can spend sometime later on to make it much better in general if you agree.